packages/graphql/src/entities/GitInfo.ts

```diff
@@ -0,0 +1,35 @@
+import { nxs, NxsResult } from 'nexus-decorators'
+import type { BaseContext } from '../context'
+import type { GitInfo as GetGitInfo } from '@packages/server/lib/util/git'
+
+@nxs.objectType({
+  description: 'Git info about a file',
+})
+export class GitInfo implements GetGitInfo {
```

Comment:
`GitInfo` will comes from CI / Cloud from #17875 and as such doesn't need to be defined here.

Reply:
I see - I thought we'd be getting git info from the repo *on your local machine*, or is this going to be a cloud only feature?

If that is the case, I'll just nuke this.

Reply:
edit, I included the wrong screenshot. This feature is for this mock. This git info is for the **spec file**, not the run.

![image](https://user-images.githubusercontent.com/19196536/133704043-af33af25-46f2-4ba9-b26a-4b638387a769.png)

Reply:
Oh ok, that looks correct then.

Reply:
Yeah in general there are a set of product features that will need to combine information from Dashboard users with information gotten locally.